src/plugins/orangehrmRecruitmentPlugin/config/routes.yaml

```
@@ -72,6 +79,22 @@ apiv2_recruitment_hiring_managers:
   defaults:
     _api: OrangeHRM\Recruitment\Api\HiringManagerAPI
 
+apiv2_recruitment_candidate_workflow:
+  path: /api/v2/recruitment/candidate-workflow/{candidateId}
+  controller: OrangeHRM\Core\Controller\Rest\V2\GenericRestController::handle
+  methods: [ GET ]
+  defaults:
+    _api: OrangeHRM\Recruitment\Api\CandidateWorkflowAPI
+    _key: id
+
+apiv2_recrutiment_reject_candidate:
+  path: /api/v2/recruitment/reject-candidate/{candidateId}
+  controller: OrangeHRM\Core\Controller\Rest\V2\GenericRestController::handle
+  methods: [ PUT ]
+  defaults:
+    _api: OrangeHRM\Recruitment\Api\RejectCandidateAPI
+    _key: id
```

```suggestion
apiv2_recrutiment_candidates_reject:
  path: /api/v2/recruitment/candidates/{candidateId}/reject
  controller: OrangeHRM\Core\Controller\Rest\V2\GenericRestController::handle
  methods: [ PUT ]
  defaults:
    _api: OrangeHRM\Recruitment\Api\RejectCandidateAPI
    _key: id
```
or
`/api/v2/recruitment/candidates/{candidateId}/actions/reject`
`/api/v2/recruitment/candidates/{candidateId}/actions/hire`

don't add these routes to this PR

---

removed. Will refactor when adding APIs